server/handlers.go

```diff
@@ -134,3 +134,81 @@ func (server HTTPServer) getRuleIDs(writer http.ResponseWriter, request *http.Re
 		return
 	}
 }
+
+// overviewEndpoint returns a map with an overview of number of clusters hit by rules
+func (server HTTPServer) overviewEndpoint(writer http.ResponseWriter, request *http.Request) {
+	authToken, err := server.GetAuthToken(request)
+	if err != nil {
+		handleServerError(writer, err)
+		return
+	}
+
+	userID := authToken.AccountNumber
+	orgID := authToken.Internal.OrgID
+
+	clustersHits := 0
+	var hitsByTotalRisk map[int]int = make(map[int]int)
+	var hitsByTags map[string]int = make(map[string]int)
+
+	clusters, err := server.readClusterIDsForOrgID(orgID)
+	if err != nil {
+		handleServerError(writer, err)
+		return
+	}
+
+	for _, clusterID := range clusters {
+		aggregatorResponse, successful := server.readAggregatorReportForClusterID(orgID, clusterID, userID, writer)
+		if !successful {
+			log.Info().Msgf("Aggregator doesn't have reports for cluster ID %s", clusterID)
+			continue
+		}
+
+		if aggregatorResponse.Meta.Count == 0 {
+			continue
+		}
+		clustersHits = clustersHits + 1
+
+		var cacheTotalRisk map[int]bool = make(map[int]bool)
```

Comment:
Didn't really get the point of these variables

Reply:
As the overview intends to show the number of clusters affected by each level of "criticality" (calculated total risk), I need to ensure that I don't count for each total risk value the same cluster twice.

This map allows me to know if this total risk has been already counted for this cluster.

OTOH, as I need to refactor this function to fulfil our cyclomatic expectations, probably I will come with a different solution and maybe I can avoid this kind of cache variables